W-17683860-failed-auto-updates (#764)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

* Incompatibility of a custom module with the latest patch version release.

[IMPORTANT]

> Using a runtime version older than three months can introduce security vulnerabilities. If an application fails to be automatically updated after two monthly patching cycles, the application is stopped and force-updated to the latest patch version. Applications that fail to start require your action to address patches. 

Reviewer: ```suggestion
Using a runtime version that is more than three months old can introduce security vulnerabilities. If your application fails to be automatically updated after two monthly patching cycles, the application is stopped and force-updated to the latest patch version. Applications that fail to start require your action to address patches. 
```
